Bio/Align/__init__.py

```diff
@@ -938,7 +938,7 @@ class Alignment:
     """
 
     @classmethod
-    def _infer_coordinates(cls, lines):
+    def infer_coordinates(cls, lines):
         """Infer the coordinates from a printed alignment."""
```

Comment:
What this method does, and what it returns, are not clear from the one line docstring.

That was OK when it was private, but we should aim higher for a public API.

Reply:
That's much better, thank you